app/popup.html

```diff
@@ -11,7 +11,8 @@
     <!-- endbuild -->
   </head>
   <body>
-    <h1>'Allo, 'Allo!</h1>
+    <h1>Made with love by</h1>
+    <p><a href="http://adorable.io/?utm_source=chrome-extension&utm_campaign=version-one-shot" target="_blank">adorable.io</a></p>
```

Comment:
Yes, UTM = Urchin Tracking Module, it just lets us know that when people arrive on adorable.io this tells us where they came from.